blockly-core/core/initialization/blockly.js

```diff
@@ -174,7 +174,18 @@ Blockly.SOUNDS_ = {};
 window.AudioContext = window.AudioContext || window.webkitAudioContext;
 
 if (window.AudioContext) {
-  Blockly.CONTEXT = new AudioContext();
+  try {
+    Blockly.AUDIO_CONTEXT = new AudioContext();
```

Comment:
Should this actually be audioContext (instead of AUDIO_CONTEXT) since it's not really a const.